completions/lintian

```diff
@@ -136,7 +136,8 @@ _lintian()
             esac
             ;;
         *)
-            _filedir '@(?(u)deb|changes|dsc)'
+            # in ubuntu, dbgsym packages end in .ddeb
+            _filedir '@(?(u|d)deb|changes|dsc|buildinfo)'
```

Comment:
Could you amend the comment with which version of lintian `*.ddeb` are supported? My 2.5.81ubuntu1 on Ubuntu 18.04 says `bad package file name [...].ddeb (neither .deb, .udeb, .changes .dsc or .buildinfo file)`

Reply:
Please have a look at the updated commit message. Thanks!